awx/main/tasks.py

```diff
@@ -1944,9 +1949,9 @@ def build_private_data(self, inventory_update, **kwargs):
             cp.add_section(section)
 
             if credential:
-                cp.set(section, 'url', credential.host)
-                cp.set(section, 'username', credential.username)
-                cp.set(section, 'password', decrypt_field(credential, 'password'))
+                cp.set(section, 'url', credential.get_input('host'))
+                cp.set(section, 'username', credential.get_input('username'))
+                cp.set(section, 'password', credential.get_input('password'))
```

Comment:
The inventory plugins transition is something that we have tried very hard to keep isolated from the rest of the code base, because the inventory source related injection logic can be kept very separate in its own corner of the code base. But if you're changing these lines - that's the one place we can't avoid conflict with.

@jladdjr is working on this now, so it's up to him. gce seems like it won't be so bad to pull in, but I don't know how far he has gone with the other sources. Pretty soon, we plan to move _all_ inventory source-related logic like this, so this is very relevant, very high-impact to the inventory plugins work and testing going on. As long as we can get it in within a day or to, I would defer to @jladdjr's judgement here.

Reply:
Yep, this is what I thought when I read this.

Given our intended schedule, I'd probably _prefer_ to err on the side of _not_ disrupting the inventory plugin work, and asking @jakemcdermott to rebase as you all start merging.

Reply:
Yep, up to you two. I'm happy do the rebasing if you two would rather not.